DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Reports/EditFiltersViewModel.cs

```diff
@@ -0,0 +1,167 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Data.Models.TrackingSystem;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using Microsoft.AspNetCore.Mvc.Rendering;
+
+    public class EditFiltersViewModel : IValidatableObject
+    {
+        private int? courseCategoryId;
+        public EditFiltersViewModel() { }
+
+        public EditFiltersViewModel(
+            ActivityFilterData filterData,
+            int userCategoryId,
+            IEnumerable<(int, string)> jobGroupOptions,
+            IEnumerable<(int, string)> courseCategoryOptions,
+            IEnumerable<(int, string)> courseOptions,
+            DateTime dataStartDate
+        )
+        {
+            JobGroupId = filterData.JobGroupId;
+
+            if (filterData.CustomisationId.HasValue)
+            {
+                FilterType = CourseFilterType.Course;
+            }
+            else
+            {
+                FilterType = filterData.CourseCategoryId.HasValue

```

Comment:
Super minor, but the course category ID check seems equivalent to the customisation if check, so I'd be tempted to do if() else if() else() structure here to emphasise that